src/Resources/config/services.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2020 Spomky-Labs
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Yoanbernabeu\AirtableClientBundle\AirtableClient;
+
+return static function (ContainerConfigurator $container): void {
+    $container = $container->services()->defaults()
+        ->private()
+        ->autoconfigure()
+        ->autowire()
```

Comment:
Great ! This will fix an error merged with PR #1

Reply:
Nice ;-)